Content.Client/UserInterface/Systems/Actions/Widgets/ActionsBar.xaml

```diff
@@ -7,7 +7,9 @@
     Orientation="Horizontal"
     HorizontalExpand="False"
 >
-    <BoxContainer Orientation="Vertical">
+    <BoxContainer
+        Orientation="Vertical"
+        Margin="0 0 0 -9"> <!-- Margin is a big hack since for some mysterious reason, the container is getting 9px added to it.... somewhere. -->
```

Comment:
I think this is just an issue with out `DefaultGameScreen` lays out controls when there is insufficient space. So instead of just fudging a margin here the layout/measure logic should just be fixed in a separate PR.

Reply:
It's been a while so I don't quite remember, but I recall this being pretty unappetizing at default res when I tested it -- either because the side bar clips into top left menu or because the arrows clip off the bottom of the screen (I don't remember which, though).  So it's going to look a bit jank at default resolution until someone gets around to fixing it properly.  

IMO it would be better to cheat it temporarily so as to not force the playerbase to have ugly ui issues as it may be some time until properly fix (unless you've heard someone is looking into this, of course).

Reply:
I generally wouldn't mind a temporary fix, but this doesn't even fix it, it still breaks when using a different UI scale or resolution, so I'd rather just not add in temporary hacks that might cause confusion later on when trying to fix the actual issue.


https://github.com/space-wizards/space-station-14/assets/60421075/0dfa6ec6-3dbb-4616-9c42-f49168f69566

Reply:
I see, thanks for the video